v2/pkg/options/options.go

```diff
@@ -47,9 +47,7 @@ type App struct {
 	AlwaysOnTop       bool
 	// BackgroundColour is the background colour of the window
 	// You can use the options.NewRGB and options.NewRGBA functions to create a new colour
-	BackgroundColour *RGBA
-	// RGBA is deprecated. Please use BackgroundColour
-	RGBA               *RGBA
+	BackgroundColour   *RGBA
```

Comment:
Should we add this as a breaking change in release notes ?

Apps working fine on 2.0.0 break on 2.1.0
@leaanthony

Reply:
Yes, though there was fair warning it was being removed. But ultimately yes, looks like it needs adding. Apologies we missed that and thanks for raising it

Reply:
No worries. Getting 2.1 to work with just a 1 line change is amazing.